x-pack/platform/plugins/shared/entity_manager/server/lib/v2/definitions/source_definition.ts

```diff
@@ -38,6 +38,7 @@ export async function storeSourceDefinition(
     index: DEFINITIONS_ALIAS,
     id: `source:${definition.source.id}`,
     document: definition,
+    refresh: 'wait_for',
```

Comment:
from https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-refresh.html

> Never start multiple refresh=wait_for requests in a row. Instead batch them into a single bulk request with refresh=wait_for and Elasticsearch will start them all in parallel and return only when they have all finished.

could we change storeSourceDefinition to accept multiple sources and bulk ?

Reply:
> I suppose it might be better if I first install all the types in parallel and use wait_for and then do all the sources in parallel with another wait_for?

yes I think two sequential bulk operations with wait_for would be more sane